easybuild/easyconfigs/d/dask/dask-2023.4.0-foss-2022b.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'PythonBundle'
+
+name = 'dask'
+version = '2023.4.0'
+
+homepage = 'https://dask.org/'
+description = """Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance
+ at scale for the tools you love."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('PyYAML', '6.0'),
+    ('SciPy-bundle', '2023.02'),
+    ('bokeh', '3.1.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('versioneer', '0.28', {
+        'checksums': ['7175ca8e7bb4dd0e3c9779dd2745e5b4a6036304af3f5e50bd896f10196586d6'],
+    }),
+    ('toolz', '0.12.0', {
+        'checksums': ['88c570861c440ee3f2f6037c4654613228ff40c93a6c25e0eba70d17282c6194'],
+    }),
+    ('locket', '1.0.0', {
+        'checksums': ['5c0d4c052a8bbbf750e056a8e65ccd309086f4f0f18a2eac306a8dfa4112a632'],
+    }),
+    ('partd', '1.4.0', {
+        'checksums': ['aa0ff35dbbcc807ae374db56332f4c1b39b46f67bf2975f5151e0b4186aed0d5'],
+    }),
+    ('fsspec', version, {
+        'checksums': ['bf064186cd8808f0b2f6517273339ba0a0c8fb1b7048991c28bc67f58b8b67cd'],
+    }),
+    ('HeapDict', '1.0.1', {
+        'checksums': ['8495f57b3e03d8e46d5f1b2cc62ca881aca392fd5cc048dc0aa2e1a6d23ecdb6'],
+    }),
+    ('zict', '3.0.0', {
+        'checksums': ['e321e263b6a97aafc0790c3cfb3c04656b7066e6738c37fffcca95d803c9fba5'],
+    }),
+    ('tblib', '1.7.0', {
+        'checksums': ['059bd77306ea7b419d4f76016aef6d7027cc8a0785579b5aad198803435f882c'],
+    }),
+    ('cloudpickle', '2.2.1', {
+        'checksums': ['d89684b8de9e34a2a43b3460fbca07d09d6e25ce858df4d5a44240403b6178f5'],
+    }),
+    (name, version, {
+        'checksums': ['33e752c1960d4987cd5aaa03c15d82df7725815771b8e21f4f6b5d4e4ec69e0e'],
+    }),
+    ('distributed', version, {
+        'checksums': ['ba54794c15e6d35ded1ee60574b29cd14004f7a6ebdee4c29dbefb546fda0010'],
+    }),
+    ('dask-mpi', '2022.4.0', {
```

Comment:
I'm surprised if this builds, i think this requires mpi4py (which was split out of SciPy-bundle). 
https://github.com/dask/dask-mpi/blob/9b6951bbb3b60246c93e5056f5b52acd4d219338/docs/environment.yml#L9

I think this should have a dependency on mpi4py-3.1.4-gompi-2022b.eb